table.go

```diff
@@ -739,6 +749,8 @@ func (t *Table[T]) DefaultMouseDown(where Point, button, clickCount int, mod Mod
 				t.selMap[id] = true
 			}
 			t.notifyOfSelectionChange()
+		case t.selMap[id]: // Sets lastClick so that on mouse up, we can treat a click and click and hold differently
+			t.lastClick = Click{id: id, check: true}
 		default: // If not already selected, replace selection with current row and make it the anchor
 			if !t.selMap[id] {
```

Comment:
Since you've added a case above that checks for `t.selMap[id]`, this if statement should be removed and its conditional content just always executed if it gets to this point.